### PR TITLE
feat(configV4): migrate config file to v4

### PR DESCRIPTION
**Pull request by paologallinaharbur**

### CODE REVIEW

File: cassandra-win-config.yml.sample [hunk 1]
```diff
@@ -1,41 +1,49 @@
...
-          # https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/entity-definition.md
-          remote_monitoring: true
-          NR_JMX_TOOL: "C:\\PROGRA~1\\NEWREL~1\\nrjmx\\nrjmx.bat"
-      labels:
-          env: production
-          role: cassandra
+integrations:
+- name: nri-cassandra
+  env:
+    port: "7199"
```
Comment (carlossscastro) [inline cassandra-win-config.yml.sample:4]: This file looks like has missed the conversion. All env vars are still lowercase with the exception of `NR_JMX_TOOL`

**Review by carlossscastro — CHANGES_REQUESTED** [review]
win config file still with lower case

File: cassandra-win-config.yml.sample (hunk 1 above)
Comment (paologallinaharbur) [reply cassandra-win-config.yml.sample]: thanks for noticing it! I am fixing it!

**Review by carlossscastro — APPROVED** [review]
Great!

**Review by carlossscastro — APPROVED** [review]
👍

**Pull request merged**